pkg/resourcemanager/predicate/ignore_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package predicate_test
+
+import (
+	resourcesv1alpha1 "github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
+	. "github.com/gardener/gardener/pkg/resourcemanager/predicate"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	gomegatypes "github.com/onsi/gomega/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+var _ = Describe("ignore", func() {
+	var (
+		managedResource *resourcesv1alpha1.ManagedResource
+		predicate       predicate.Predicate
+	)
+
+	BeforeEach(func() {
+		managedResource = &resourcesv1alpha1.ManagedResource{}
+	})
+
+	Describe("#NotIgnored", func() {
+		BeforeEach(func() {
+			predicate = NotIgnored()
+		})
+
+		Context("#Create", func() {
+			It("should match because no ignore mode annotation", func() {
```

Comment:
`s/ignore mode/ignore/`

```suggestion
			It("should match because no ignore annotation", func() {
```